dolphinscheduler-dao/src/main/resources/sql/upgrade/3.2.0_schema/mysql/dolphinscheduler_dml.sql

```diff
@@ -30,6 +30,9 @@ END IF;
 END;
 d//
 
+-- If the admin account is not associated with a tenant, the admin's tenant will be set to the default tenant.
+UPDATE `t_ds_user` SET `tenant_id` = '-1' WHERE (`user_name` = 'admin') AND (`tenant_id` = '0');
+
```

Comment:
```suggestion
-- If the admin account is not associated with a tenant, the admin's tenant will be set to the default tenant.
UPDATE `t_ds_user` SET `tenant_id` = '-1' WHERE `tenant_id` = '0';

```

Reply:
Not recommended:
'''
UPDATE t_ds_user SET tenant_id = '-1' WHERE tenant_id = '0';
'''

Response：
Error Code: 1175. You are using safe update mode and you tried to update a table without a WHERE that uses a KEY column. To disable safe mode, toggle the option in Preferences -> SQL Editor and reconnect.

Reply:
@ruanwenjun 
Can we just check admin's tenant to ensure that admin can log in normally, and let the user set or adjust his own tenant after logging in?